packages/docs-theme/src/styles/_layout.scss

```diff
@@ -37,63 +35,112 @@ Flex utils
 Page layout elements
 */
 
-.docs-root {
-  background-color: $content-background-color;
-
-  // usually this element is _way_ longer than the viewport, but sometimes it isn't.
-  // this ensures that it'll always at least fill the viewport so white page doesn't show through.
-  min-height: 100vh;
+.docs-app {
+  display: flex;
+  flex-direction: row;
+  background-color: $sidebar-background-color;
+  height: 100vh;
 
   &.pt-dark {
-    background-color: $dark-content-background-color;
+    background-color: $dark-sidebar-background-color;
   }
 }
 
-.docs-app {
-  // center container in window
-  margin: auto;
-  width: $container-width;
-  padding-top: $pt-navbar-height;
+.docs-nav-wrapper,
+.docs-content-wrapper {
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  flex-shrink: 1;
+  overflow: auto;
+}
+
+.docs-nav-wrapper {
+  flex-basis: $sidebar-width;
+  align-items: flex-end;
+  position: relative;
+  box-shadow: 1px 0 0 $pt-divider-black;
+
+  .pt-dark & {
+    box-shadow: 1px 0 0 $pt-dark-divider-black;
+  }
 }
 
-// navbar fixed to top of screen with same width as rest of app
-.docs-navbar {
-  position: fixed;
-  top: 0;
-  width: inherit;
+.docs-content-wrapper {
+  flex-basis: $content-width;
+  align-items: flex-start;
+  background-color: $content-background-color;
+
+  .pt-dark & {
+    background-color: $dark-content-background-color;
+  }
 }
 
-.docs-content {
-  // leave space for nav menu on the left (position: fixed)
-  margin-left: $sidebar-width;
-  padding: 0 $sidebar-padding $content-padding $content-padding;
+.docs-page {
+  max-width: $content-width;
+  padding: 0 0 $content-padding $content-padding * 2;
 }
 
 .docs-nav {
-  position: fixed;
-  top: $pt-navbar-height;
-  bottom: 0;
   width: $sidebar-width;
-  overflow: auto;
-  padding: $content-padding $sidebar-padding;
+
+  .docs-navigator {
+    margin-bottom: $pt-grid-size * 2;
+    width: 100%;
+    padding: 0 $pt-grid-size / 2;
+  }
+
+  > * {
+    padding: $sidebar-padding;
+  }
+}
+
+.docs-nav-title {
+  @include pt-flex-container(row);
+  flex-wrap: wrap;
+  align-items: center;
 }
 
 /*
 Lefthand navigation menu
 */
 
-// nested menus are hidden by default
-.docs-nav-menu .docs-nav-menu {
-  display: none;
-}
+.docs-nav-menu {
+  .pt-menu-item {
+    padding-right: $pt-grid-size;
+    padding-left: 0;
+    white-space: initial;
 
-.docs-nav-expanded > .docs-nav-menu {
-  display: block;
-}
+    &:hover,
+    &.pt-active,
+    &.docs-nav-expanded {
+      // bold text instead of background color
+      background-color: transparent !important; // stylelint-disable-line declaration-no-important
+      font-weight: 600;
```

Comment:
unfortunate side effect of this is that it can change the number of lines which an item takes up. so it can push down the layout of the rest of the nav tree. a minor inconvenience 🤷‍♂️

Reply:
yeah I noticed as well, I'll update that but maybe not in this PR (unless I figure something great before this merges)